ietf/submit/forms.py

```diff
@@ -758,6 +758,20 @@ def cleaned_line(self):
             line = formataddr((line, email))
         return line
 
+    def clean_name(self):
+        name = super(SubmitterForm, self).clean_name()
+        if name.startswith('=?'):
+            msg = f'"{name}" appears to be a MIME-encoded string.'
+            try:
+                import email.header
+                text, encoding = email.header.decode_header(name)[0]
+                decoded_name = text.decode(encoding)
+                msg += f' Did you mean "{decoded_name}"?'
```

Comment:
Do you know whether the user pasting in the MIME-encoded string sees their original with the `=?` characters, or does their browser render it as decoded? (If it's the latter, I'm wondering if their experience will be a field containing "Jane Doe" with a warning "Did you mean 'Jane Doe'?" leading to confusion)

Reply:
It appears that the literal string "=?utf-8?q?Beno=C3=AEt_Viguier?=" was pasted into the submitter field. In any case, that's what I'm checking for, and the error string would look like `"=?utf-8?q?Jane_Doe?=" appears to be a MIME-encoded string. Did you mean "Jane Doe"?`